src/Transport/CircuitBreakers/RepeatedFailuresOverTimeCircuitBreaker.cs

```diff
@@ -40,7 +40,7 @@ public Task Failure(Exception exception)
                 Logger.WarnFormat("The circuit breaker for {0} is now in the armed state", name);
             }
 
-            return Task.Delay(TimeSpan.FromSeconds(1));
+            return TaskEx.Completed;
```

Comment:
Different PR. I've already made a mistake by fixing it here, which insight I shouldn't have done on this PR :)